### ISSUE TITLE
Use in combination with SoundSwitch

### ISSUE THREAD

**Issue by iampxyco** [erikrichardlarson/unbox#55]
I last tried to use my light control and Unbox at the same time, unfortunately it doesn't work.
Unbox picks up the data first and therefore disrupts the lighting control.
When I quit Unbox it works, when I start it again the error reappears.
Is this bug known, will it be fixed in the future or is it not possible to use both at the same time?

Thanks for the great tool!

**Comment by erikrichardlarson** [comment]
Yeah this is a known bug, and unfortunately there isn't a fix at the moment. I'd love to test some more once I can get my hands on denon gear and SoundSwitch, I'll be sure to update you here if I'm able to test this some more!
